### PR TITLE
Add mnemonics to frequently used dialogs

### LINKED ISSUE #9192: Consistent selection highlight vs. focus
## Current behaviour

1. `RevisionGrid` and tab `Diff`: look active if unfocused

2. Left Panel: always shows a selection, which is irrelevant and may be misleading if unfocused

3. FormPush: local and remote branch ComboBoxes look focused. It is almost invisible whether the click to focus worked or not. 

## Expected behaviour

1. & 3. should be handled as in the tab `File tree` (and the current Left Panel)
2. Left Panel selection should be hidden if unfocused

## Steps to reproduce

- hit <kbd>Ctrl</kbd>+<kbd>0</kbd>
- select a branch
- hit <kbd>Ctrl</kbd>+<kbd>1</kbd>
- hit <kbd>Down</kbd> -> Left Panel selection not related
- hit <kbd>Ctrl</kbd>+<kbd>3</kbd>
- hit <kbd>Ctrl</kbd>+<kbd>3</kbd>
- hit <kbd>Ctrl</kbd>+<kbd>4</kbd>
- hit <kbd>Ctrl</kbd>+<kbd>4</kbd>
- hit <kbd>Ctrl</kbd>+<kbd>Up</kbd>

### To be improved

image
image

### Good

image
image

## Did this work in previous version of GitExtensions

no

## Environment

- Git Extensions 33.33.33
- Build 559bb84c999be456c4e0e4cec1a8709aae9e78cf
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET 5.0.5
- DPI 96dpi (no scaling)

### PR DESCRIPTION
**Pull request by mstv**
## Proposed changes

- Add mnemonics to
  - `FormCheckoutRevision`
  - `FormCherryPick`
  - `FormRevertCommit`
  - `FormPush`
- `FormPush`: Avoid `ComboBox`es appearing focused after `ResizeDropDownWidth`
  Fixes #9192 3.

## Screenshots 
Before|After
-|-
grafik|grafik
grafik|grafik
grafik|grafik
grafik|grafik
grafik|grafik

## Test methodology 

- manual

## Test environment(s) 

- Git Extensions 33.33.33
- Build 607a1b4bc424fa1667bf532b3c4b3c1827661cb5
- Git 2.31.1.windows.1
- Microsoft Windows NT 10.0.19042.0
- .NET 5.0.6
- DPI 96dpi (no scaling)

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

File: GitUI/CommandsDialogs/FormPush.Designer.cs [hunk 1]
```diff
@@ -93,7 +93,7 @@ private void InitializeComponent()
             this.ckForceWithLease.Name = "ckForceWithLease";
             this.ckForceWithLease.Size = new System.Drawing.Size(109, 17);
             this.ckForceWithLease.TabIndex = 0;
-            this.ckForceWithLease.Text = "Force With &Lease";
+            this.ckForceWithLease.Text = "Force with &lease";
```
Comment (gerhardol) [inline GitUI/CommandsDialogs/FormPush.Designer.cs:96]:
```suggestion
            this.ckForceWithLease.Text = "&Force with lease";
```

File: GitUI/CommandsDialogs/FormPush.Designer.cs [hunk 2]
```diff
@@ -104,7 +104,7 @@ private void InitializeComponent()
             this.ForcePushBranches.Name = "ForcePushBranches";
             this.ForcePushBranches.Size = new System.Drawing.Size(79, 17);
             this.ForcePushBranches.TabIndex = 1;
-            this.ForcePushBranches.Text = "&Force Push";
+            this.ForcePushBranches.Text = "&Force push";
```
Comment (gerhardol) [inline GitUI/CommandsDialogs/FormPush.Designer.cs:107]: (if this is the site close to force-with-lease)
`F` should be used with force-with-lease.
There was an issue requesting that --force was removed completely. I still prefer to keep it but usage should not be prominent
```suggestion
            this.ForcePushBranches.Text = "Force push";
```

**Review by gerhardol — APPROVED** [review]
with a comment

File: GitUI/CommandsDialogs/FormPush.Designer.cs (hunk 2 above)
Comment (mstv) [reply GitUI/CommandsDialogs/FormPush.Designer.cs]: `F` is used with force-with-lease now (and `O` for "F&orce push" because it would look incomplete).

**Pull request merged**